### PR TITLE
Fix jump clicking select multiselect field

### LINKED ISSUE #2115: [Panel] Dropdown "jumps" on select
**Describe the bug**
Using the multiselect field type with a list of years, the dropdown "jumps" to the top of the list when selecting an option from the end of the list. It looks as if the dropdown is re-rendered on select.

**To Reproduce**
This is our blueprint, the API return a list of years:

```yml
years:
  type: multiselect
  label: Zeitraum
  sort: true
  options: api
  api:
    url: '{{ kirby.url }}/hananils/years'
    fetch: years
  translate: false
```

**Expected behavior**
The dropdown scroll position should not change. Otherwise it's very hard to select consecutive options.

**Screenshots**
Before selecting:
grafik

After selecting:
grafik

**Kirby Version**
Latest release and current `develop` branch. 

**Desktop**
Latest Firefox on macOS Mojave.

### LINKED ISSUE #2135: [Panel] e.nextSibling.focus is not a function on multiselect field
**Describe the bug**  
Error occured when navigate right (no next siblings) selected options on multiselect field

**To Reproduce**  
1. Go to panel
2. Add multiselect field with options to any page
3. Edit page and select some options
4. Navigate selected options with `right` arrow

**Screenshots**  
siblings

Screenshot actions: TAB + RIGHT + LEFT + RIGHT + RIGHT

**Kirby Version**  
3.2.5 Stable

**Console output**  
```
app.js:9365 TypeError: e.nextSibling.focus is not a function
    at a.navigate (app.js:4900)
    at n.nativeOn.keydown (app.js:4698)
    at oe (vendor.js:41)
    at HTMLSpanElement.n (vendor.js:41)
    at HTMLSpanElement.Ni.i._wrapper (vendor.js:41)
```

**Desktop**  
 - Windows 10
 - Chrome 77

**Additional context**  
 - Left arrow key working properly when no prev siblings
 - Left and right arrow actions working properly on tags fields

### PR DESCRIPTION
**Pull request by afbora**
## Describe the PR

First I added click.prevent to the clicked element, its parent elements, and related fields, but it didn't work. In my research, I found that this problem can be solved by save the scroll position and I have implemented such a solution.

Save scroll position of selected item before selection to prevent scroll top (#2115)

I wanted to fix the other error as there was a minor update in the same file (#2135)

## Related issues

- Fixes #2115
- Fixes #2135 

## Todos

- [ ] Add unit tests for fixed bug/feature
- [x] Pass all unit tests
- [ ] Fix code style issues with CS fixer and `composer fix`
- [ ] If needed, in-code documentation (DocBlocks etc.)

### CODE REVIEW

**Review by bastianallgeier — COMMENTED** [review]
Unfortunately this doesn't solve the problem for me.

**Comment by afbora** [comment]
Sorry, my fault 😞 I thought it was a simple click prevent error, but it wasn't as easy as I expected. I haven't been able to find the source yet, but after the RC release I'll take time again.

File: panel/src/components/Forms/Input/MultiselectInput.vue [hunk 1]
```diff
@@ -202,12 +203,12 @@ export default {
...
       switch (direction) {
         case "prev":
-          if (current && current.previousSibling) {
+          if (current && current.previousSibling && current.previousSibling.focus) {
             current.previousSibling.focus();
           }
           break;
         case "next":
-          if (current && current.nextSibling) {
+          if (current && current.nextSibling && current.nextSibling.focus) {
```
Comment (afbora) [inline panel/src/components/Forms/Input/MultiselectInput.vue:211]: Fixes #2135 with checking focus property

**Comment by afbora** [comment]
PR description updated

**Review by bastianallgeier — APPROVED** [review]
Works like a charm ❤️

**Pull request merged**